services/core-api/app/api/parties/response_models.py

```diff
@@ -41,6 +47,21 @@
         'documents': fields.Nested(MINE_DOCUMENT)
     })
 
+NOW_APPLICATION_MODEL = api.model('NOWApplication', {
+    'now_application_guid': fields.String,
+    'now_number': fields.String,
+    'submitted_date': Date,
```

Comment:
Maybe fields.Date ?

Reply:
We use a pre made date field that allows for the proper formatting of our dates.